File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/DocumentUnits/DocumentUnitFilterModel.cs
using VecompSoftware.DocSuiteWeb.Model.Entities.Fascicles;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.DocumentUnits
{
    public class DocumentUnitFilterModel
    {
        public DocumentUnitFilterModel() { }

        public FascicleModel Fascicle { get; set; }

        public string DocumentUnitName { get; set; }

        public ReferenceType? ReferenceType { get; set; }

        public string Title { get; set; }
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/DocumentUnits/DocumentUnitFilterModel.ts
import FascicleReferenceType = require('App/Models/Fascicles/FascicleReferenceType');
import FascicleModel = require('App/Models/Fascicles/FascicleModel');

interface DocumentUnitFilterModel {
    Fascicle: FascicleModel;
    DocumentUnitName: string;
    ReferenceType?: FascicleReferenceType
    Title: string;
}

export = DocumentUnitFilterModel;